pkg/services/ngalert/store/alert_rule.go

```diff
@@ -416,35 +415,36 @@ func (st DBstore) GetNamespaceByUID(ctx context.Context, uid string, orgID int64
 	return folder, nil
 }
 
-func (st DBstore) getFilterByOrgsString() string {
+func (st DBstore) getFilterByOrgsString() (string, []interface{}) {
 	if len(st.Cfg.DisabledOrgs) == 0 {
-		return ""
+		return "", nil
 	}
 	builder := strings.Builder{}
 	builder.WriteString("org_id NOT IN(")
 	idx := len(st.Cfg.DisabledOrgs)
+	args := make([]interface{}, 0, len(st.Cfg.DisabledOrgs))
 	for orgId := range st.Cfg.DisabledOrgs {
-		builder.WriteString(strconv.FormatInt(orgId, 10))
+		args = append(args, orgId)
+		builder.WriteString("?")
```

Comment:
`?` is not compatible with Postgres, so we need to do something different depending on dialect. Can we use Gorm?

Reply:
Xorm takes care of replacing `?` according to the dialect.